homeapp/templates/base.html

```diff
@@ -162,16 +162,16 @@ <h4 class="modal-title">Login with</h4>
                         <div class="flex-caption">
                             {% block header-text %}
                             {% endblock %}
-                            <form method="GET" action="{% url 'searchapp:search' %}"
+                            <form method="GET" action="{% url 'search:search' %}"
                                   class="form-inline row input-group" id="search">
+                                {% csrf_token %}
```

Comment:
Is it necessary for this token to be placed before all other inputs?
can it not be place down where I had placed it earlier??